src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -47,7 +48,16 @@ public static function create(\Exception $exception, $statusCode = null, array $
             $statusCode = 500;
         }
 
+        switch ($statusCode) {
+            case 404:
+                $title = 'Sorry, the page you are looking for could not be found.';
```

Comment:
"route", "resource" and "endpoint" are very technical terms that most regular users wont understand.

I prefer "URL" (I agree with [this WHATWG spec](https://url.spec.whatwg.org/#goals) that the terms URI and IRI are confusing and not widely used).

Inspired by Google's 404 page I suggest the following:
"Sorry, the requested URL was not found on this server."

I think this is neutral, to the point, and does not use too much tech lingo.